ddtrace/contrib/tornado/compat.py

```diff
@@ -1,16 +0,0 @@
-try:
```

Comment:
Because of the way that futures is checked for. Importing from another contrib should be considered an anti-pattern. Contribs should only be allowed to import from generic utilities, but not within each other.